catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -311,7 +328,36 @@ public AttributeFormat getAttributeFormat() {
                 false /* tokenized */,
                 true /* multivalued */,
                 STRING_TYPE));
-        return descriptors;
+        DESCRIPTORS = Collections.unmodifiableSet(descriptors);
+
+        BASIC_METACARD = new MetacardTypeImpl(MetacardType.DEFAULT_METACARD_TYPE_NAME,
```

Comment:
Either way your going to have to create an instance copy all of its descriptors into your type in order to "mixin" lets just skip that first step.